tethys_apps/migrations/0001_initial_20.py

```diff
@@ -56,6 +56,7 @@ class Migration(migrations.Migration):
                                                               parent_link=True, primary_key=True, serialize=False,
                                                               to='tethys_apps.TethysAppSetting')),
                 ('value', models.CharField(blank=True, max_length=1024)),
+                ('default', models.CharField(blank=True, default='', max_length=1024)),
```

Comment:
Don't modify migration files directly. Instead run `python manage.py makemigrations` to create a new migration and commit it. Otherwise, existing installations won't be able to get updated.